paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/activity/PaymentSheetPlaygroundActivity.kt

```diff
@@ -452,7 +452,7 @@ class PaymentSheetPlaygroundActivity : AppCompatActivity() {
         val checkboxLabel = if (viewBinding.shippingAddressCheckboxLabel.text.isNotBlank()) {
             viewBinding.shippingAddressCheckboxLabel.text.toString()
         } else {
-            getString(R.string.stripe_paymentsheet_address_element_same_as_shipping)
+            getString(R.string.same_as_shipping)
```

Comment:
I think it's typically best practice to prefix these for SDKs so we don't overwrite app developers resources.